src/permissions.rs

```diff
@@ -32,7 +32,9 @@ impl DenoPermissions {
       return Ok(());
     };
     // TODO get location (where access occurred)
-    let r = permission_prompt("Deno requests access to run a subprocess.");
+    let r = permission_prompt(
+      "👨‍👧  Deno requests access to run a subprocess.",
```

Comment:
What do you think about this symbol ⤴️?

Reply:
@daynin I think 🔀 is better for this

Reply:
It looks like processes swap some data or something else. I think, it may confuse users

Reply:
@daynin Could you write you suggestion as Unicode emoji, not GH one?

Reply:
@sh7dm yes, of course

Subprocess: https://emojipedia.org/arrow-pointing-rightwards-then-curving-upwards/
Environment: https://emojipedia.org/personal-computer/ or https://emojipedia.org/desktop-computer/

Reply:
thanks